### PR TITLE
 SCC reader output length in percentage units

### LINKED ISSUE #86: SCC reader should not output length in cell units
IMSC prohibits the use of cell units but for `ebutts:linePadding` -- see https://www.w3.org/TR/ttml-imsc1.1/#length-cell

Instead SCC reader should use percentages.

### PR DESCRIPTION
**Pull request by valnoel**
Closes #86

### CODE REVIEW

File: src/main/python/ttconv/scc/utils.py [hunk 1]
```diff
@@ -44,3 +45,25 @@ def get_extent_from_dimensions(width: Union[int, Number], height: Union[int, Num
...
+
+
+def convert_cells_to_percentages(dimensions: Union[PositionType, ExtentType], cell_resolution: CellResolutionType) -> Union[
+  PositionType, ExtentType]:
+  """Converts dimensions from cell units to percentage units"""
+  if isinstance(dimensions, PositionType):
+    if dimensions.x.units is not LengthType.Units.c or dimensions.y.units is not LengthType.Units.c:
+      raise ValueError("Dimensions to convert must be in cell units")
+
+    x_percent = dimensions.x.value * 100 / cell_resolution.columns
```
Comment (palemieux) [inline src/main/python/ttconv/scc/utils.py:57]: Percentages should be rounded to the nearest integer.

**Review by palemieux — APPROVED** [review]

File: src/main/python/ttconv/scc/utils.py [hunk 2]
```diff
@@ -44,3 +45,25 @@ def get_extent_from_dimensions(width: Union[int, Number], height: Union[int, Num
...
+    y_percent = dimensions.y.value * 100 / cell_resolution.rows
```
Comment (palemieux) [inline src/main/python/ttconv/scc/utils.py:58]: Percentages should be rounded to the nearest integer

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/python/ttconv/scc/utils.py (hunk 2 above)
Comment (valnoel) [reply src/main/python/ttconv/scc/utils.py]: Fixed in 17ee6eb

File: src/main/python/ttconv/scc/utils.py (hunk 1 above)
Comment (valnoel) [reply src/main/python/ttconv/scc/utils.py]: Fixed in 17ee6eb

**Review by palemieux — APPROVED** [review]

**Pull request merged**